content_scripts/vimium_frontend.coffee

```diff
@@ -336,13 +336,8 @@ window.focusThisFrame = do ->
   # Inject stylesheet.
   _styleSheet = document.createElement "style"
   if _styleSheet.style?
-    _styleSheet.innerHTML = ""
+    _styleSheet.innerHTML = "@import url(\"#{chrome.runtime.getURL("content_scripts/vimium.css")}\");"
```

Comment:
This looks straightforward, @mrmr1993.

Is there a reason for using `\"` and not just `'` for the outer quotes?